src/aat/resources/features/F-029/F-029.feature

```diff
@@ -0,0 +1,66 @@
+@F-029
+Feature: F-029: Fetch an event trigger in the context of a case type
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-049
+  Scenario: must return successfully a CaseEventTrigger object filtered by case type ID
```

Comment:
it would be useful to use a language more close to the business than the code for certain elements. For example here `CaseEventTrigger` is not part of the business language. A BA or QA reading this won't understand what a 'trigger' is. That terminology is only used in our codebase. I would change this to:

`must return successfully the current case event data filtered by case type ID`
or something similar